src/signature-request/expiration-datetime.ts

```diff
@@ -0,0 +1,26 @@
+import * as t from "io-ts";
+
+import * as E from "fp-ts/lib/Either";
+import { pipe } from "fp-ts/lib/function";
+
+import { isBefore } from "date-fns";
+
+const isDate = (m: unknown): m is Date => m instanceof Date;
+export const FutureDate = new t.Type<Date, string, unknown>(
```

Comment:
I think that the "is" part should handle the expiration as well otherwise 
`FutureDate.is(<some time in the past>)` will pass (can you add some unit tests about this?)

probably this type is overkill (moreover, it's a little bit odd it changes in time) and an inline check with `isBefore` suffices.

Reply:
I agree. Better not have this type, I'll change the validation